### PR TITLE
Updating the documentation for dd-dotnet

### PR DESCRIPTION
**Pull request by kevingosse**
### What does this PR do? What is the motivation?

Update the diagnostic doc to mention the new dd-dotnet tool. Update the expected outputs.

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/troubleshooting/dotnet_diagnostic_tool.md [hunk 1]
```diff
@@ -48,22 +58,51 @@ No issue found with the target process.
...
 
 Running checks on process 35888
-Target process is running with .NET Framework
-Profiler is not loaded into the process
-Tracer is not loaded into the process
-The environment variable COR_ENABLE_PROFILING should be set to '1' (current value: '0')
-```
+Process name: SimpleApp
+
+---- STARTING TRACER SETUP CHECKS -----
```
Comment (andrewlock) [inline content/en/tracing/troubleshooting/dotnet_diagnostic_tool.md:66]: Where are the "issues"? 🤔

File: content/en/tracing/troubleshooting/dotnet_diagnostic_tool.md [hunk 2]
```diff
@@ -48,22 +58,51 @@ No issue found with the target process.
 
 Example output with issues:
 ```bash
-$ dd-trace check process 35888
+$ dd-dotnet check process 35888
```
Comment (andrewlock) [inline content/en/tracing/troubleshooting/dotnet_diagnostic_tool.md:61]: I think you need to update the previous example too

**Review by andrewlock — COMMENTED** [review]
LGTM, except I think you missed an update/updated one with the wrong grab

**Review by link04 — CHANGES_REQUESTED** [review]
It looks like the example session the process check is using the older No Issues output instead of the new one and the output with issues has no issues, for the IIS output they both  got no issues. Could we add an example where they haven't set a key for it to show up on the `---- STARTING TRACER SETUP CHECKS -----` section?

**Comment by kevingosse** [comment]
> It looks like the example session the process check is using the older No Issues output instead of the new one and the output with issues has no issues, for the IIS output they both got no issues. Could we add an example where they haven't set a key for it to show up on the `---- STARTING TRACER SETUP CHECKS -----` section?

My bad, fixed in https://github.com/DataDog/documentation/pull/20734/commits/ca9e467639d64b2f0ff5d43ff9b0128382f31bc4

**Review by andrewlock — APPROVED** [review]
LGTM thanks

**Review by link04 — APPROVED** [review]

**Review by urseberry — APPROVED** [review]

**Pull request merged**